buttons-examples/iOS/ios-button.md

```diff
@@ -1,10 +1,12 @@
 <!--docs:
-title: "Buttons"
+title: ttons"<component name>"
 layout: detail
 section: components
-excerpt: "Buttons allow users to take actions. and make choices, with a single tap."
+excerpt: "iOS Buttons"
+ide_version: "<cIDE name> <compatible IDE version and build number>"
+material_package_version: "<compatible Material platform package version number>"
 iconId:
-path: /catalog/buttons
```

Comment:
We need to change this to "/catalog/FABs" so the script that pulls this page into Material.io has a separate page for FABs as per the design schema.

So:
https://material.io/develop/ios/components/buttons/

Gets divided into:
https://material.io/develop/ios/components/buttons/
https://material.io/develop/ios/components/FAB/

Reply:
But since this is the Buttons doc shouldn't it be `/catalog/buttons`? https://github.com/mingjane-work/doc-material-components/pull/65 is the FABas PR, and it's like `/catalog/FAB/` in that one. Should it be `/catalog/FABs` there?

Reply:
Oops, you're right: I was getting confused between the FAB and the Buttons docs when I was making these comments.